process/coordinator/process.go

```diff
@@ -676,9 +672,6 @@ func (tc *transactionCoordinator) CreateMarshalizedData(body block.Body) (map[ui
 
 		interimProc := tc.getInterimProcessor(miniblock.Type)
 		if !check.IfNil(interimProc) {
-			if !appended {
```

Comment:
With this mechanism we assured that we can not call both the preprocessor and interimprocessor for the same miniblock. It is ok as it is now?

Reply:
yes it is as the marshalizing of the body is done in another place. here you do only marshalizing of the transactions.